system-functionalities.md

```diff
@@ -338,14 +338,40 @@ Implementation guidance: For security, all verification checks run by the key se
 The asset owner can create and use signing keys in a manner similar to arbitrary secrets, with the additional operation of creating signatures.
 
 ### Inherited Operations
-All of the [protocols for arbitrary secrets](#operations-on-arbitrary-secrets) should be supported. The only difference is the type of key created and used. That is, calling the generation functionality should allow the asset owner to create one of the following key types:
+All of the [protocols for arbitrary secrets](#operations-on-arbitrary-secrets) should be supported. The only significant difference is the type of key created and used; minor protocol modifications are described below. That is, calling the generation functionality should allow the asset owner to create one of the following key types:
 - ECDSA on secp256 curve; or
 - Ed25519 (i.e., EdDSA on edwards25519).
 
-Signing keys have an additional supported operation, namely, the creation of a signature.
+Signing keys have an additional supported operation, namely, the creation of a signature. They also have an additional public component that can be derived from a generated key.
```

Comment:
This is too vague. When adjusting the generation function descriptions, first define more clearly what the signing key type is. Then remove this sentence.

Reply:
Updated the generation intro. I didn't copy the same text to import, do you think it would be helpful there as well?